leaderelection/src/main/java/io/scalecube/leaderelection/RaftProtocol.java

```diff
@@ -0,0 +1,10 @@
+package io.scalecube.leaderelection;
+
+/**
+ * Created by ronenn on 9/13/2016.
+ */
+public class RaftProtocol {
+  public static final String RAFT_GOSSIP_HEARTBEAT = "raft.protocol.heartbeat";

```

Comment:
Don't create separate empty class for constants, move them into protocol file and also follow qualifier pattern same as for other system messages.